src/modules/arangodb/README.md

```diff
@@ -0,0 +1,52 @@
+# ArangoDB
+
+ArangoDB is an open source frienly multi-model database.
+
+## Examples
+
+Jest test to verify the connection
```

Comment:
i wonder if we should include the full test example, or just the different ways in which the container can be used like 
```javascript
new ArandoDBContainer().start()
```
the two reasons i mention it is that whoever is looking at the README will see the test file right there, and secondly this README is a copy of the test file, so why worry about keeping two things in sync?

Reply:
You are right. I think it is best to link the test. ArangoDB is well documented. I linked the documentation too.